.circleci/config.yml

```diff
@@ -866,6 +866,30 @@ jobs:
           command: yarn type:doc
       - store_artifacts:
           path: ./docs/out
+  
+  # Node core language tests
+
+  node-core-language-tests:
```

Comment:
`node-core-` prefix is for jobs running tests on the core of the tracer.

Reply:
Perhaps `node-upstream-tests`?